addons/core/addon/styles/addon.scss

```diff
@@ -65,3 +65,10 @@
     overflow: hidden;
   }
 }
+
+// Utility classes
+.container-inputs__margin-bottom {
```

Comment:
I put this in `core` for now but likely should just be admin, I was thinking of making a separate CSS file just for utility classes, any thoughts?

Reply:
how is this different from our usual overrides that exist in rose/hds/overrides.scss?

Reply:
The idea is these aren't overrides nor CSS for specific pages, but generic classes that are limited in scope and just apply some specific CSS which can be composable. I'm generally wary of using too many utility classes but I've noticed we've done the same or nearly the same styles for some things like margin and then duplicated them across css classes that are specific to pages.

Reply:
thanks for the explanation. Agree that we should clean up those specific duplicated margins and having a separate CSS for utility classes sounds like a good idea!